docs/.vitepress/config.ts

```diff
@@ -28,6 +29,7 @@ export default defineConfig({
         text: 'Effects',
         items: [
           { text: 'Bloom', link: '/guide/effects/bloom' },
+          { text: 'Depth of Field', link: '/guide/effects/depth-of-field' },
```

Comment:
Just asking, is there other category than "effects" in post-processing pkg?

Reply:
There is only "Guide" (Introduction) and "Effects" right now.

Reply:
@JaimeTorrealba EffectComposer will only have abstractions that are in fact effects/passes, there is no notorious categorizing like in `cientos` abstractions